packages/contracts-ops/contracts/Config.sol

```diff
@@ -80,14 +82,18 @@ abstract contract Config {
             );
     }
 
-    function home(string memory domain)
+    function homeUpgrade(string memory domain)
         public
         onlyInitialized
         returns (Upgrade memory)
     {
         return abi.decode(loadCoreAttribute(domain, "home"), (Upgrade));
     }
 
+    function home(string memory domain) public onlyInitialized returns (Home) {
```

Comment:
if you're adding these for Home and Replica, should also add for GovernanceRouter, BridgeRouter, and TokenRegistry?

Reply:
yes, as they become needed